src/integration-tests/bash/run.sh

```diff
@@ -3122,6 +3126,18 @@ function test_suite_init {
     declare_new_test 1 "$@"
 
     cd $PROJECT_ROOT || fail "Could not cd to $PROJECT_ROOT"
+ 
+    # Handle '--show-all' deprecation for 'kubectl get jobs' and 'kubectl get pods'.
+    # --show-all yields a deprecation warning in stderr in 1.10 in later, since
+    # it isn't needed in 1.10 and later.
+    # TBD It's not clear if we should check the Client version or the Server version.
```

Comment:
I suspect we just care about the server version.

Reply:
Looks like its the client version, so the code can stay unchanged.   The Wercker runs use a 1.10 client and a 1.9 server, and tests are passing.  This indicates the 1.10 client gets 'show all' behavior by default even though the server itself is older.  I've pushed a change with the TBD comment removed.